hugo/layouts/partials/create_description.html

```diff
@@ -0,0 +1,59 @@
+<!-- creates the site meta description for each page, good for search engine optimization -->
+
+{{ $description := "" }}
+
+{{ if .IsHome }}
+    {{ $description = "A platform facilitating access and discovery of genome data of non-model eukaryotic species studied in Sweden, hosted by SciLifeLab" }}
+
+<!-- Individ pages -->
+{{ else if eq .Title "Contact" }}
+    {{ $description = "Contact us for more information and support." }}
+
+{{ else if eq .Title "About" }}
+    {{ $description = "Learn more about the Swedish Reference Genome Portal and its mission." }}
+
+{{ else if eq .Title "Information på svenska" }}
+    {{ $description = "Lär dig mer om den Swedish Reference Genome Portal och sin uppdrag." }}
```

Comment:
Small suggestion to make the Swedish sound a tad bit more formal:

Lär dig mer om Swedish Reference Genome Portal och dess syfte.

Reply:
I'm glad you checked that one, changed it with commit ba9a44d41fd262d707af4c031b6573c9d68f9507